src/external/visualization_integration.cuh

```diff
@@ -3,19 +3,20 @@
 
 
 #include "../../lib/HTTPRequest/include/HTTPRequest.hpp"
+#include <string>
+#include <vector>
 
 #include "../simulation_objects/MapNode.cuh"
-#include <string>
 
 
 /**
  * Reads the first line from the file containing the visualization endpoint url and returns it
  *
- * Path to the file (relative to the executable's location) is `config/visualization_endpoint.txt`
+ * Path to the file (relative to the executable's location) is `ч`
  *
  * @returns Url to send data to be visualized to
  */
-__host__ std::string get_visualization_endpoint();
+__host__ std::pair<std::string, std::string> get_visualization_endpoint();
```

Comment:
1. Shouldn't it be `get_visualization_endpoints`?
2. The doc should be updated (the `@returns` section is outdated)